packages/fluentui/react-bindings/src/hooks/useTriggerElement.tsx

```diff
@@ -33,7 +33,7 @@ export function useTriggerElement(props: UseTriggerElementOptions): React.ReactE
   const trigger = childrenExist(props.children) ? props.children : props.trigger;
   const element = trigger ? (React.Children.only(trigger) as React.ReactElement) : null;
 
-  if (process.env.NODE_ENV !== 'production') {
+  if (process.env.NODE_ENV !== 'production' && process.env.NODE_ENV !== 'test') {
```

Comment:
```suggestion
  // An exception should not be thrown in tests as components might be rendered without styles
  if (process.env.NODE_ENV !== 'production' && process.env.NODE_ENV !== 'test') {
```

Reply:
@layershifter tests got broken because it explicit expects to thrown, what do you suggest?

Reply:
Should be possible to change the value of NODE_ENV and clean it up explicitly for those tests. Otherwise we'd need to keep this change as a patch in TMP

Reply:
You can obviously pass `.env` for tests

```tsx
  const OLD_ENV = process.env.NODE_ENV;

  beforeEach(() => {
    process.env.NODE_ENV = 'development'
  });

  afterAll(() => {
    process.env.NODE_ENV = OLD_ENV 
  });

```

Reply:
Applied the suggestion